console/src/app/pages/projects/apps/app-create/app-create.component.html

```diff
@@ -381,6 +381,14 @@ <h3 class="title cnsl-secondary-text">
             </span>
           </span>
         </div>
+        <div class="row">
+          <span class="left cnsl-secondary-text">
+            {{ 'APP.SAML.CERTIFICATE' | translate }}
+          </span>
+          <a href="/saml/v2/certificate" class="cert" target="_blank" rel="noreferrer">{{
```

Comment:
Can we construct the link using the issuer from the environment.json?

Reply:
Sure!